### PR TITLE
add explicit deleted flag to slack user info

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

**Comment by eishay** [comment]
@ryanpbrewster

File: kifi-backend/modules/common/app/com/keepit/slack/models/SlackAuth.scala [hunk 1]
```diff
@@ -174,6 +175,7 @@ object SlackUserInfo {
     (__ \ 'id).read[SlackUserId] and
     (__ \ 'name).read[SlackUsername] and
     (__ \ 'profile).read[SlackUserProfile] and
+    (__ \ 'deleted).read[Boolean] and
```
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/slack/models/SlackAuth.scala:178]: Are you sure you don't want this to be `readNullable[Boolean].map(_.contains(true))`?

Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/slack/models/SlackAuth.scala:178]: I guess are we sure it's always in the JSON?

Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/slack/models/SlackAuth.scala:178]: not sure, good idea

**Pull request closed without merging**